Sources/CCryptoBoringSSL/CMakeLists.txt

```diff
@@ -319,80 +252,21 @@ add_library(CCryptoBoringSSL STATIC
   "crypto/x509/x_spki.c"
   "crypto/x509/x_val.c"
   "crypto/x509/x_x509.c"
-  "crypto/x509/x_x509a.c")
+  "crypto/x509/x_x509a.c"
+  "gen/crypto/err_data.c")
 
 if(CMAKE_SYSTEM_NAME STREQUAL Darwin AND CMAKE_SYSTEM_PROCESSOR MATCHES "amd64|x86_64")
   target_sources(CCryptoBoringSSL PRIVATE
-    crypto/chacha/chacha-x86_64-mac.mac.x86_64.S
```

Comment:
What happened to all of the asm? And is this big `if` needed anymore?

Reply:
Good catch. It's still there, but I suspect the script is looking in the wrong place.

Reply:
Ah yeah, so these are now generated by BoringSSL, and it expects the names to match.

Reply:
Now fixed.

Reply:
OOI what happens if they are missing? Presumably there's a lower-perf fallback?

Reply:
No, I think the compile would fail. You need `NO_ASM` set to fallback. We don't build the cmake version in CI here which is why we didn't notice.

Reply:
There is a `cmake` CI job which did pass:
https://ci.swiftserver.group/job/swift-crypto-cmake-prb/306/ 

It builds the project with ninja: https://github.com/apple/swift-crypto/blob/81bee98e706aee68d39ed5996db069ef2b313d62/docker/docker-compose.yaml#L37

and it looks like we only set `NO_ASM` when building was WASI:
https://github.com/apple/swift-crypto/blob/81bee98e706aee68d39ed5996db069ef2b313d62/Package.swift#L104

This is a little surprising given the above...

Reply:
Yeah, that's _very_ curious.